playbooks/edx_sandbox.yml

```diff
@@ -53,6 +52,7 @@
       celery_worker: True
     - edxapp
     - ecommerce
```

Comment:
Needs to add the `role:` label to allow the `when:` clause to be added:
```yaml
- role: ecommerce
  when: SANDBOX_ENABLE_ECOMMERCE
```

Reply:
@pomegranited Ack, sorry about the back-and-forth here. I've been trying to squeeze this in while focusing on something else, which *I know* is not the best idea, even for small changes like this... 😑 Anyway, d65cc437641b985099c27ca2399aa6bd47d71e45 should do what we need.